docs/source/exporting.rst

```diff
@@ -48,8 +48,6 @@ http://nbconvert.readthedocs.io/en/latest/api/exporters.html
 
 .. autoclass:: LenvsHTMLExporter
```

Comment:
yes, true, but this has actually already been removed in #794, so doesn't need doing again here